tests/integration/labs/single.yaml

```diff
@@ -0,0 +1,218 @@
+---
+lab:
+  description: ""
+  notes: ""
+  title: cisco.ios.ios
+  version: 0.2.2
+links:
+  - id: l0
+    n1: n0
+    n2: n1
+    i1: i1
+    i2: i0
+    conditioning: {}
+    label: cat8000v-0-GigabitEthernet1<->ext-conn-0-port
+nodes:
+  - boot_disk_size: null
+    configuration: |2-
+      Building configuration...
+
+      Current configuration : 3276 bytes
+      !
+      ! Last configuration change at 08:17:20 UTC Tue Nov 5 2024
+      !
+      version 17.13
+      service timestamps debug datetime msec
+      service timestamps log datetime msec
+      platform qfp utilization monitor load 80
+      platform punt-keepalive disable-kernel-core
+      platform sslvpn use-pd
+      platform console serial
+      !
+      hostname cisco
+      !
+      boot-start-marker
+      boot-end-marker
+      !
+      !
+      no aaa new-model
+      !
+      !
+      !
+      !
+      !
+      !
+      !
+      !
+      !
+      !
+      !
+      !
+      ip domain name ansible.com
+      !
+      !
+      !
+      login on-success log
+      !
+      !
+      subscriber templating
+      !
+      pae
+      !
+      !
+      crypto pki trustpoint SLA-TrustPoint
+       enrollment pkcs12
+       revocation-check crl
+       hash sha256
+      !
+      !
+      crypto pki certificate chain SLA-TrustPoint
```

Comment:
Do we need to push this as part of the configuration?
Can we not just add the best needed configuration?